### ISSUE TITLE
[Web-UI] UX visualization Improvements

### ISSUE THREAD

**Issue by do-me** [qdrant/qdrant#3361]
The Web-UI (`http://localhost:6333/dashboard#/collections/<collection>/visualize`) is very useful for quick exploration of a collection.
However, the UX might need some minor improvements. I guess that the most common use case is free exploration i.e., you take a look at your points and see which topics seem to form some kind of cluster. By doing so, currently when you hover over the points, first the raw JSON is displayed. Displaying the embeddings here is not needed.
![image](https://github.com/qdrant/qdrant/assets/47481567/261dc47e-c75e-4d92-95e9-c163ccfe0276)

When you click on the point, an overlay appears in a nice tabular format. The embeddings are hidden by default.
![image](https://github.com/qdrant/qdrant/assets/47481567/ee0c2fa1-bac2-42a3-be2b-a61a61187588)

I think what would be better suited is an integration of the overlay table in the right side bar as there is lots of available space. Maybe two tabs would be good, one for the request and one for the point info?
![image](https://github.com/qdrant/qdrant/assets/47481567/906bdb12-015a-452f-a89a-b64f3c556fd6)

If the tabular format was displayed for each point on hover in the right side bar, one could much easier explore a collection. 

Anyway thanks a lot for providing the tool - much appreciated! :)

**Comment by Snehil-Shah** [comment]
![image](https://github.com/qdrant/qdrant/assets/130062020/9cc1659d-2f83-4eb8-925b-14244369df6d)

The embeddings are not visible for me on hover, only the payload. Are embeddings part of your payload in your example?

**Comment by do-me** [comment]
Yes indeed they were - thanks for the hint! The JSON overlay looks already much better this way.
The only problem I have with it - especially when you have overlapping points - is that you sort of lose track of either the mouse/point you queried and the meta info you're interested in, e.g. the title as the overlay "jumps" up/down when there is more than one entry.

**Comment by Snehil-Shah** [comment]
Yes, your solution is definitely cleaner. When selecting a point, we can highlight that point in the visualization and a table in the sidebar can show the info of the selected/highlighted point.

**Comment by do-me** [comment]
I don't want to feature creep but that would also allow for a possible batch selection with the mouse, e.g. drawing a rectangle/free polygon and selecting all points within. I think such a feature could possibly be useful when you'd like to create topic clusters manually. Based on the clustered points you could calculate an average embedding and then have some ground for automatic classification.

A bit off-topic but is there a way I could somehow export the 2D-coordinates along with the metadata somehow? I guess all the variables/processed data are kept isolated in the worker.js right? 

I would like to visualize the data in a different tool for some tests.
